### DISCUSSION TITLE
X-Application-Id

### DISCUSSION THREAD

**Discussion by ErbenSamiloglu** [go-kit/kit#1275] [Ideas]
Hello,

I'm using GoKit for one of my applications and need to use an application id request header to identify integrated applications. Can we add a simple  header into http package or is there any idea to do this?

I've done this in my local environment and if this is OK, I can open a pull request for this type of change.

![image](https://github.com/go-kit/kit/assets/125267229/7287ee04-1677-42ad-b85c-19ae50bf3ee0)

**Comment by peterbourgon** [comment]
No reason to do this in Go kit, you can do it in your application just fine.
